src/app/main/component/comments/components/comment-textarea/comment-textarea.component.ts

```diff
@@ -267,6 +283,29 @@ export class CommentTextareaComponent implements OnInit, AfterViewInit, OnChange
     });
   }
 
+  private updateLinksInTextarea(currentText: string): void {
+    if (Patterns.urlLinkifyPattern.test(currentText)) {
+      this.commentTextarea.nativeElement.innerHTML = this.renderLinks(currentText);
```

Comment:
You are using innerHTML to render the links, which could expose the application to XSS
A safer approach might be to use Angular's DomSanitizer